vllm/config.py

```diff
@@ -1161,6 +1171,9 @@ def __init__(
             typical_acceptance_sampler_posterior_alpha (Optional[float]):
                 A scaling factor for the entropy-based threshold in the
                 TypicalAcceptanceSampler.
+            disable_logprobs: If set to True, token log probabilities will
+                not be output in both the draft worker and the target worker.
```

Comment:
nit: we also disable logprobs calculation in the SpecDecodeWorker. Suggest:
```
If set to True, token log probabilities will not be returned even if requested by
sampling parameters. This reduces latency by skipping logprob calculation
in proposal sampling, target sampling, and after accepted tokens are determined.

```

Reply:
Done.